synfig-core/src/synfig/threadpool.h

```diff
@@ -43,6 +43,7 @@
 
 /* === C L A S S E S & S T R U C T S ======================================= */
 
+using namespace synfig;
```

Comment:
[Never use `using namespace` in a header file](https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers).